src/author/views.py

```diff
@@ -373,8 +372,7 @@ def dispatch(self, request, *args, **kwargs):
         )
 
     def get_form_kwargs(self):
-        """Renders the email body and subject for editing using the form
-        """
+        """Renders the email body and subject for editing using the form."""
```

Comment:
I believe we agreed on having a space after the quotes here, but I might be wrong.